app/src/main/java/com/github/se/gomeet/ui/authscreens/register/RegisterNameCountryPhone.kt

```diff
@@ -102,56 +109,52 @@ fun RegisterNameCountryPhone(
             keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
             modifier = Modifier.fillMaxWidth())
 
-        Spacer(modifier = Modifier.size(screenHeight / 60))
-
-        Column {
-          TextField(
-              value = country,
-              onValueChange = {
-                country = it
-                filteredCountries =
-                    if (it.isEmpty()) {
-                      countries.value
-                    } else {
-                      countries.value.filter { c ->
-                        c.lowercase(Locale.getDefault())
-                            .startsWith(it.lowercase(Locale.getDefault()))
-                      }
-                    }
-                expanded = true
-              },
-              label = { Text("Select Country") },
-              singleLine = true,
-              colors = textFieldColors,
-              readOnly = false,
-              modifier = Modifier.fillMaxWidth())
-          DropdownMenu(
-              expanded = expanded,
-              onDismissRequest = { expanded = false },
-              offset =
-                  DpOffset(
-                      x = 20.dp, y = 20.dp), // Adjusts the position directly below the TextField
-              modifier =
-                  Modifier.fillMaxWidth()
-                      .heightIn(max = 200.dp) // Limits the height to display around 5 items
-              ) {
-                for (c in filteredCountries) {
-                  DropdownMenuItem(
-                      text = { Text(c) },
-                      onClick = {
-                        country = c
-                        expanded = false
-                      })
-                }
-              }
-        }
-
-        if (!countryValid && !firstClick) {
-          Text(text = "Country is not valid", color = Color.Red)
+        if (!validLastName && !firstClick) {
+          Text(text = "Last Name is not valid", color = Color.Red)
         }
 
         Spacer(modifier = Modifier.size(screenHeight / 60))
-
+        /*Column {
```

Comment:
Need to remove all this commented out code

Reply:
My mistake! This is fixed.